From 3d485859812ee743007ec8ac73a40c3f4e0431a7 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <yasufum.o@gmail.com>
Date: Tue, 4 Jun 2019 16:02:45 +0900
Subject: [PATCH 2/2] shared/sec: rename command_conn.c

This patch is to rename `command_conn.c` and `command_conn.h`, and
update Makefiles.

Signed-off-by: Yasufumi Ogawa <yasufum.o@gmail.com>
---
 src/mirror/Makefile                                             | 2 +-
 src/pcap/Makefile                                               | 2 +-
 src/pcap/cmd_runner.c                                           | 2 +-
 src/shared/secondary/spp_worker_th/cmd_runner.c                 | 2 +-
 .../secondary/spp_worker_th/{command_conn.c => conn_spp_ctl.c}  | 2 +-
 .../secondary/spp_worker_th/{command_conn.h => conn_spp_ctl.h}  | 0
 src/vf/Makefile                                                 | 2 +-
 7 files changed, 6 insertions(+), 6 deletions(-)
 rename src/shared/secondary/spp_worker_th/{command_conn.c => conn_spp_ctl.c} (99%)
 rename src/shared/secondary/spp_worker_th/{command_conn.h => conn_spp_ctl.h} (100%)

diff --git a/src/mirror/Makefile b/src/mirror/Makefile
index d1ae9ce..3e31cfa 100644
--- a/src/mirror/Makefile
+++ b/src/mirror/Makefile
@@ -19,7 +19,7 @@ SRCS-y += ../shared/common.c
 SRCS-y += ../shared/secondary/utils.c ../shared/secondary/add_port.c
 SRCS-y += $(SPP_WKT_DIR)/cmd_utils.c
 SRCS-y += $(SPP_WKT_DIR)/spp_port.c
-SRCS-y += $(SPP_WKT_DIR)/command_conn.c
+SRCS-y += $(SPP_WKT_DIR)/conn_spp_ctl.c
 SRCS-y += $(SPP_WKT_DIR)/cmd_parser.c
 SRCS-y += $(SPP_WKT_DIR)/cmd_runner.c
 SRCS-y += $(SPP_WKT_DIR)/string_buffer.c
diff --git a/src/pcap/Makefile b/src/pcap/Makefile
index 61f6f81..c3d5ae4 100644
--- a/src/pcap/Makefile
+++ b/src/pcap/Makefile
@@ -18,7 +18,7 @@ SRCS-y := spp_pcap.c
 SRCS-y += cmd_utils.c
 SRCS-y += cmd_runner.c cmd_parser.c
 SRCS-y += ../shared/common.c
-SRCS-y += $(SPP_WKT_DIR)/command_conn.c
+SRCS-y += $(SPP_WKT_DIR)/conn_spp_ctl.c
 SRCS-y += $(SPP_WKT_DIR)/spp_port.c
 SRCS-y += $(SPP_WKT_DIR)/ringlatencystats.c
 SRCS-y += $(SPP_WKT_DIR)/string_buffer.c
diff --git a/src/pcap/cmd_runner.c b/src/pcap/cmd_runner.c
index 9541cbb..8a1fb87 100644
--- a/src/pcap/cmd_runner.c
+++ b/src/pcap/cmd_runner.c
@@ -9,7 +9,7 @@
 
 #include "shared/secondary/spp_worker_th/string_buffer.h"
 #include "spp_pcap.h"
-#include "shared/secondary/spp_worker_th/command_conn.h"
+#include "shared/secondary/spp_worker_th/conn_spp_ctl.h"
 #include "cmd_parser.h"
 #include "cmd_runner.h"
 
diff --git a/src/shared/secondary/spp_worker_th/cmd_runner.c b/src/shared/secondary/spp_worker_th/cmd_runner.c
index 92d727b..1d1ec49 100644
--- a/src/shared/secondary/spp_worker_th/cmd_runner.c
+++ b/src/shared/secondary/spp_worker_th/cmd_runner.c
@@ -13,7 +13,7 @@
 #include "spp_port.h"
 #include "string_buffer.h"
 
-#include "command_conn.h"
+#include "conn_spp_ctl.h"
 #include "cmd_parser.h"
 #include "cmd_runner.h"
 
diff --git a/src/shared/secondary/spp_worker_th/command_conn.c b/src/shared/secondary/spp_worker_th/conn_spp_ctl.c
similarity index 99%
rename from src/shared/secondary/spp_worker_th/command_conn.c
rename to src/shared/secondary/spp_worker_th/conn_spp_ctl.c
index b80c54b..030da92 100644
--- a/src/shared/secondary/spp_worker_th/command_conn.c
+++ b/src/shared/secondary/spp_worker_th/conn_spp_ctl.c
@@ -14,7 +14,7 @@
 
 #include "shared/common.h"
 #include "string_buffer.h"
-#include "command_conn.h"
+#include "conn_spp_ctl.h"
 
 #define RTE_LOGTYPE_SPP_COMMAND_PROC RTE_LOGTYPE_USER1
 
diff --git a/src/shared/secondary/spp_worker_th/command_conn.h b/src/shared/secondary/spp_worker_th/conn_spp_ctl.h
similarity index 100%
rename from src/shared/secondary/spp_worker_th/command_conn.h
rename to src/shared/secondary/spp_worker_th/conn_spp_ctl.h
diff --git a/src/vf/Makefile b/src/vf/Makefile
index bdeff98..faf72ee 100644
--- a/src/vf/Makefile
+++ b/src/vf/Makefile
@@ -18,7 +18,7 @@ SRCS-y := spp_vf.c classifier_mac.c spp_forward.c
 SRCS-y += $(SPP_WKT_DIR)/string_buffer.c
 SRCS-y += $(SPP_WKT_DIR)/ringlatencystats.c
 SRCS-y += $(SPP_WKT_DIR)/spp_port.c
-SRCS-y += $(SPP_WKT_DIR)/command_conn.c
+SRCS-y += $(SPP_WKT_DIR)/conn_spp_ctl.c
 SRCS-y += $(SPP_WKT_DIR)/cmd_parser.c
 SRCS-y += $(SPP_WKT_DIR)/cmd_runner.c
 SRCS-y += $(SPP_WKT_DIR)/cmd_utils.c
-- 
2.17.1

